webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/queries/LdapQueryProcessor.scala

```diff
@@ -386,7 +433,8 @@ class InternalLDAPQueryProcessor(
       }.map { case (ldapFilters, specialFilters) =>
         val finalLdapFilter = normalizedQuery.composition match {
           case Or  =>
-            Some(OR( (filterSeqSet.foldLeft(ldapFilters)( _ union _ )).toSeq:_*))
```

Comment:
that used to create a syntaxically false filter that trigger an ldap error (server side) and make open ldap returns nothings